src/components/FilterBarInline/FilterBarInline.mdx

```diff
@@ -380,6 +402,52 @@ return (
   language="jsx"
 />
 
+## TagSet Example
+
+<Canvas of={FilterBarInlineStories.TagSetExample} />
+
```

Comment:
Can you add a short explanation to say that the following code is just one example of how to use it but it could be done differently depending on the app's needs?